x/oracle/types/denom.go

```diff
@@ -30,3 +30,13 @@ func (dl DenomList) String() (out string) {
 
 	return strings.TrimSpace(out)
 }
+
+// Contains checks whether or not a SymbolDenom (e.g. UMEE) is in the DenomList
+func (dl DenomList) Contains(symbolDenom string) bool {
+	for _, d := range dl {
+		if strings.ToUpper(d.SymbolDenom) == strings.ToUpper(symbolDenom) {
```

Comment:
nit: I've realized there's a more efficient way of doing these comparisons:

https://pkg.go.dev/strings#EqualFold